.pre-commit-config.yaml

```diff
@@ -11,3 +11,17 @@ repos:
     hooks:
       - id: yamllint
         args: [-c, .yamllint.yaml]
+  - repo: https://github.com/PyCQA/flake8
+    rev: 7.0.0
+    hooks:
+      - id: flake8
+        args:
+          - --max-line-length=120
```

Comment:
Probably should put this in a config file so we can have an editor pick it up to help folks do the right thing.